docs/en/blog/posts/2024-06-01-documenting-rust-code-with-markdown.md

```diff
@@ -0,0 +1,200 @@
+---
+author: Kristijan Lenković
+authors:
+  - klenkovic
+date: 2024-06-01
+readtime: 15
+tags:
+  - rust
+  - markdown
+  - documentation
+  - rustdoc
```

Comment:
I suggest removing `rustdoc` and `axum` since these tags are unlikely to be repeated in future posts, but `rust` should be kept